### PR TITLE
RSDK-7980: Gripper component

### PR DESCRIPTION
**Pull request by lia-viam**
See [RSDK-7980](https://viam.atlassian.net/browse/RSDK-7980)

[RSDK-7980]: https://viam.atlassian.net/browse/RSDK-7980?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/viam/sdk/components/private/gripper_server.cpp [hunk 1]
```diff
@@ -0,0 +1,68 @@
+#include <viam/sdk/components/private/gripper_server.hpp>
+
+#include "gripper_server.hpp"
```
Comment (stuqdog) [inline src/viam/sdk/components/private/gripper_server.cpp:3]: I think probably we should drop this `include` line?

File: src/viam/sdk/components/private/gripper_server.cpp [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    return make_service_helper<Gripper>("GripperServer::Open", this, request)(
+        [&](auto& helper, auto& gripper) { gripper->open(helper.getExtra()); });
+}
+
+::grpc::Status GripperServer::Grab(
+    ::grpc::ServerContext*,
+    const ::viam::component::gripper::v1::GrabRequest* request,
+    ::viam::component::gripper::v1::GrabResponse* response) noexcept {
+    return make_service_helper<Gripper>("GripperServer::Grab", this, request)(
+        [&](auto&, auto& gripper) { response->set_success(gripper->grab()); });
```
Comment (stuqdog) [inline src/viam/sdk/components/private/gripper_server.cpp:25]: we should set `extra` in the `gripper->grab(...)` call.

**Review by stuqdog — APPROVED** [review]
A couple small things (no need to re-request review) otherwise lgtm!

File: src/viam/sdk/components/private/gripper_server.cpp (hunk 1 above)
Comment (lia-viam) [reply src/viam/sdk/components/private/gripper_server.cpp:3]: ugh yes these keep getting put in automatically somehow not sure which tool is doing it 😠

**Pull request merged**